api/organisations/invites/tests/test_views.py

```diff
@@ -96,12 +96,16 @@ def test_update_invite_link_returns_405(invite_link, admin_client, organisation)
 
 
 def test_join_organisation_with_permission_groups(
-    test_user, test_user_client, organisation, user_permission_group
+    test_user, test_user_client, organisation, user_permission_group, subscription
 ):
     # Given
     invite = Invite.objects.create(email=test_user.email, organisation=organisation)
     invite.permission_groups.add(user_permission_group)
 
+    # update subscription to add another seat
+    subscription.max_seats = 2
```

Comment:
This value shouldn't be used anymore, right? We should be using the value from Chargebee so I'm not sure how updating it here is making a difference to this test?

Reply:
Well, only if it is a Chargebee subscription - we still fallback to this if it's not